tests/sql/moz-fx-data-shared-prod/org_mozilla_fenix_nightly_derived/geckoview_version_v1/metrics.schema.json

```
@@ -0,0 +1,39 @@
+[
+  {
+    "type": "TIMESTAMP",
+    "name": "submission_timetamp",
+    "mode": "REQUIRED"
+  },
+  {
+    "type": "RECORD",
+    "name": "client_info",
+    "mode": "NULLABLE",
+    "fields": [
+      {
+        "type:": "STRING",
+        "name": "app_build",
+        "mode": "NULLABLE"
+      }
+    ]
+  },
+  {
+    "type": "RECORD",
+    "name": "metrics",
+    "mode": "NULLABLE",
+    "fields": [
+      {
+        "type": "RECORD",
+        "name": "string",
+        "mode": "NULLABLE",
+        "fields": [
+          {
+            "type:": "STRING",
```

there's a misplaced colon in the type key
```suggestion
        "type": "STRING",
        "name": "app_build",
        "mode": "NULLABLE"
      }
    ]
  },
  {
    "type": "RECORD",
    "name": "metrics",
    "mode": "NULLABLE",
    "fields": [
      {
        "type": "RECORD",
        "name": "string",
        "mode": "NULLABLE",
        "fields": [
          {
            "type": "STRING",
```

---

I definitely would not have caught that on my own, thanks!